Sources/tart/Credentials/DockerConfigCredentialsProvider.swift

```diff
@@ -25,6 +25,8 @@ class DockerConfigCredentialsProvider: CredentialsProvider {
 
     let process = Process.init()
     process.executableURL = executableURL
+    // pass environment variables so things like AWS_PROFILE are picked up
+    process.environment = ProcessInfo.processInfo.environment
```

Comment:
I think this is a no-op:

>By default, a Process object inherits its environment from the process that launches it.

(from https://developer.apple.com/documentation/foundation/process)

Reply:
My bad. Seems user should just switch to newer `credHelpers`.